src/app/tests/suites/certification/Test_TC_S_2_2.yaml

```diff
@@ -20,273 +20,692 @@ PICS:
-      disabled: true
+      command: "RecallScene"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+              - name: "SceneID"
+                value: 0x01
 
     - label:
           "TH configures AC1 on DUT for all implemented application clusters
           supporting scenes."
-      verification: |
-          DUT is configured with AC1.
-      disabled: true
+      cluster: "Level Control"
+      command: "MoveToLevelWithOnOff"
+      arguments:
+          values:
+              - name: "Level"
+                value: 100
+              - name: "TransitionTime"
+                value: 0
+              - name: "OptionsMask"
+                value: 0
+              - name: "OptionsOverride"
+                value: 0
 
     - label:
           "TH sends a StoreScene command to DUT with the GroupID field set to
           0x0001 and the SceneID field set to 0x01."
       PICS: S.S.C04.Rsp
-      verification: |
-          DUT sends a StoreSceneResponse command to TH with the Status field set to 0x00 (SUCCESS), the GroupID field set to 0x0001 and the SceneID field set to 0x01.
-      disabled: true
+      command: "StoreScene"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+              - name: "SceneID"
+                value: 0x01
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "GroupID"
+                value: 0x0001
+              - name: "SceneID"
+                value: 0x01
 
     - label:
           "TH configures AC2 on DUT for all implemented application clusters
           supporting scenes."
-      verification: |
-          DUT is configured with AC2 which is different to AC1.
-      disabled: true
+      cluster: "Level Control"
+      command: "MoveToLevelWithOnOff"
+      arguments:
+          values:
+              - name: "Level"
+                value: 200
+              - name: "TransitionTime"
+                value: 0
+              - name: "OptionsMask"
+                value: 0
+              - name: "OptionsOverride"
+                value: 0
 
     - label:
           "TH sends a StoreScene command to DUT with the GroupID field set to
           0x0001 and the SceneID field set to 0x02."
       PICS: S.S.C04.Rsp
-      verification: |
-          DUT sends a StoreSceneResponse command to TH with the Status field set to 0x00 (SUCCESS) or 0x89 (INSUFFICIENT_SPACE). If SUCCESS, with the GroupID field set to 0x0001 and the SceneID field set to 0x02. If INSUFFICIENT_SPACE, the following steps SHALL NOT be executed.
-      disabled: true
+      command: "StoreScene"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+              - name: "SceneID"
+                value: 0x02
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
```

Comment:
Test plan allows for INSUFFICIENT_SPACE here.

Reply:
The test plan allows INSUFFICIENT_SPACE but states that the following steps shall not be executed. Given that yaml doesn't currently handle if statements, how should we handle an insufficient space result here?

Reply:
@vivien-apple Is there a way to condition steps on our saveAs value so we know whether to expect INSUFFICIENT_SPACE here?

Reply:
Kept to 0x00 for now, should we add a TODO on this?

Reply:
There should be an issue tracking the fact that we can't do the right thing here, yes, and a comment pointing to that issue.

Reply:
I have made https://github.com/project-chip/connectedhomeip/pull/26567 to fulfil the request. An example of how to condition a step based on some output of a previous step can be found in `src/app/tests/suites/TestEqualities.yaml`